.github/workflows/webflux.yml

```diff
@@ -53,6 +53,7 @@ jobs:
                     - webflux-couchbase
                     - webflux-couchbase-es-oauth2
                     - webflux-psql
+                    - webflux-nodb-oauth2
```

Comment:
We already have a lot of integration tests, we can't test all combinations on each PR and merge.
I don't think this one deserves to be tested (even for daily-builds).

Reply:
I've put here to fix https://github.com/jhipster/generator-jhipster/issues/11620.

> I don't think this one deserves to be tested (even for daily-builds).

If so we should forbid reactive + 'no' databaseType.
CC @mraible

Reply:
No, we don't need to forbid it. 
But we can't automatically test all possible combinations of all the options we provide, that would be too much. 
And this is why a few combination are tested here, and a few others there: https://github.com/hipster-labs/jhipster-daily-builds

Reply:
Yep, I was planning to put the test there.
I just think fixing without adding a test to daily build seems wrong.

Reply:
Removed.